fixdeps.sh

```diff
@@ -1,3 +1,6 @@
-cp -rf ./node_modules/@react-native-community/* ./packages/identity-wallet-mobile/node_modules/@react-native-community
+# cp -rf ./node_modules/@react-native-community/* ./packages/identity-wallet-mobile/node_modules/@react-native-community
+cp -rf ./node_modules/@react-native-community ./packages/identity-wallet-mobile/node_modules/
```

Comment:
seems strange, is there a way to install the module inside the workspace as well without copying it?

Reply:
Yes, the nohoist option in the pacakge.json should do it (this is configured there). However, this is not working properly. I should put a comment on this file about that since this is a workaround. Is it possible to fix it in a separate issue?

Reply:
sure